### PR TITLE
closes #22

### LINKED ISSUE #22: anyway to add a separator between arbitrary rows?
would be handy to be able to somehow indicate that there is a separator between two arbitrary rows.

perhaps if a row was not an array but instead the keyword `:separator`  ??

(FYI this is a very nice gem! thanks!)

### PR DESCRIPTION
**Pull request by catmando**
adds ability to specify which rows get separators (issue #22)

### Requirements
[x] Tests written & passing locally?
[x] Code style checked?  
[] Rebased with `master` branch?
[x] Documentation updated?

### CODE REVIEW

**Comment by piotrmurach** [comment]
Hi Mitch,

Thank you so much for this! I had a quick look and all looks good.

I'm currently finishing couple of things in another project but once done I will merge and release. Please bear with me.

File: README.md [hunk 1]
```diff
@@ -542,6 +542,65 @@ end
...
+#  |header1|header2|
+#  +-------+-------+
+#  |a1     |a2     |
+#  |b1     |b2     |
+#  +-------+-------+
+#  |c1     |c2     |
+#  +-------+-------+
+```
+
+**Note:** if you supply a detailed list of rows to separate, then the separator between the header and the rows
```
Comment (piotrmurach) [inline README.md]: Could please explain why is that the preferred default behaviour?

**Comment by catmando** [comment]
no rush

**Comment by piotrmurach** [comment]
Thank you! ❤️

**Pull request merged**